app/javascript/hyacinth_ui_v1/components/projects/enabled_dynamic_fields/EnabledDynamicFieldForm.js

```diff
@@ -273,10 +289,15 @@ class EnabledDynamicFieldEdit extends React.Component {
   }
 
   render() {
-    const { dynamicFieldGraphs, disabled } = this.state;
+    const {
+      dynamicFieldGraphs, disabled,
```

Comment:
The `disabled` state value seems to be derived from the `formType` props value, which never changes as far as I can tell.  I know you didn't add this, so this is more of a suggestion for the future.

Could we either just use formType, or pass in disabled as a prop?  In the PermissionsEditor component, I pass in a `readOnly` prop that has the same effect as `disabled`, and I'd propose that as a good naming alternative.  

Generally though, it would be simpler to check a boolean rather than a string value (show vs. edit), assuming we only ever plan to have two modes.